packages/aws-cdk/lib/logging.ts

```diff
@@ -7,6 +7,11 @@ const { stdout, stderr } = process;
 
 type WritableFactory = () => Writable;
 
+// LOG_LOCK is an array rather than a boolean because it needs to be modified by other
+// parts of the CLI and imported variables are always immutable.
+export const LOG_LOCK = [false];
```

Comment:
this is almost certainly the hackiest thing i've ever come up with. whats the better way to do something like this?

Reply:
ok i thought of a better way immediately lol